djangocms_text_ckeditor/fields.py

```diff
@@ -12,28 +15,69 @@
     pass
 
 
+class HTMLFormField(CharField):
+
+    widget = TextEditorWidget
+
+    def __init__(self, *args, **kwargs):
+        conf = kwargs.pop('configuration', None)
+
+        if conf:
+            widget = TextEditorWidget(configuration=conf)
+        else:
+            widget = None
+        kwargs.setdefault('widget', widget)
+        super(HTMLFormField, self).__init__(*args, **kwargs)
+
+    def clean(self, value):
+        value = super(HTMLFormField, self).clean(value)
+        return clean_html(value, full=False)
+
+
 class HTMLField(models.TextField):
     configuration = None
 
+    if LTE_DJANGO_1_7:
+        __metaclass__ = models.SubfieldBase
```

Comment:
I think that this is not Python 3 compatible. 
Python 3 uses ``metaclass=`` kwarg on class declaration. 
Alternatively ``six.with_metaclass`` (http://pythonhosted.org/six/#six.with_metaclass) can be used to maintain Python 2 and Python 3 compatibility.